Fails when using NPM v7

These are my scripts in package.json:

```json
"test": "is-ci \"test:coverage\" \"test:watch\"",
"test:watch": "jest --watch",
"test:coverage": "jest --coverage"
```

Running `npm test` fails in npm v7 (npm@7.4.1,  node@15.5.1):

```sh
$ npm test

> vk-group-tracking-bot@0.6.0 test
> is-ci "test:coverage" "test:watch"

npm ERR! code 1
npm ERR! path /home/lynxtaa/code/vk-group-tracking-bot
npm ERR! command failed
npm ERR! command sh -c is-ci "test:coverage" "test:watch"
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -16,7 +16,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [8, 10, 12, 14]
+        node-version: [8, 10, 12, 14, 15]
         os: [ubuntu-latest, windows-latest, macos-latest]
 
     steps:

--- a/README.md
+++ b/README.md
@@ -16,7 +16,7 @@ In your package.json:
 
 ```js
 "scripts": {
-  "test": "is-ci test:ci test:local"
+  "test": "is-ci-cli test:ci test:local"
 }
 ```
 

--- a/package.json
+++ b/package.json
@@ -5,7 +5,8 @@
   "license": "MIT",
   "repository": "YellowKirby/is-ci-cli",
   "bin": {
-    "is-ci": "cli.js"
+    "is-ci": "cli.js",
+    "is-ci-cli": "cli.js"
   },
   "engines": {
     "node": ">=8"

